content/en/docs/concepts/components.md

```diff
@@ -75,7 +75,7 @@ For more information, see
 
 {{% docs/languages/exporters/intro %}}
 
-### Automatic Instrumentation
+### Zero-Code Instrumentation
```

Comment:
Is this the official name we'll use for automatic instrumentation then? Could it be "Zero-code" (non capitalized "Code")?

Reply:
I have no strong preference, we can go with "zero-code"

Reply:
I was wondering if this requires a... <suspense music> discussion? https://github.com/open-telemetry/opentelemetry.io/discussions

Reply:
We have that zero code discussion already, so we might add a few words there if you like

Reply:
Right, forgot about that one...

Reply:
I am fine with this different terminology, but we should clarify that there's an official term for it as well.

Reply:
> I am fine with this different terminology, but we should clarify that there's an official term for it as well.

Official means "automatic"?

Reply:
@cartermp can you give me some details what you'd like to have clarified? Do you think we need some words about "automatic" and "manual" still and giving some context why the docs prefer a different wording or something?